src/java.desktop/share/classes/javax/swing/JTabbedPane.java

```diff
@@ -722,7 +722,7 @@ public void setSelectedComponent(Component c) {
      * @param component the component to be displayed when this tab is clicked.
      * @param tip the tooltip to be displayed for this tab
      * @param index the position to insert this new tab
-     *       ({@code > 0 and <= getTabCount()})
+     *       ({@code >= 0 and <= getTabCount()})
      *
      * @throws IndexOutOfBoundsException if the index is out of range
```

Comment:
Seems fine. The existing text actually contradicts the exception text immediately below.

Even though it is therefore arguably a typo, I suggest even so that a CSR be filed for this.